### DISCUSSION TITLE
Bigger file preview size

### DISCUSSION THREAD

**Discussion by lluishi93** [directus/directus#13026] [Feature Requests]
It'd be nice if the file preview was bigger in order to see it better.

Current preview:
![image](https://user-images.githubusercontent.com/14786440/165505092-2b7cd40b-f1d0-4289-805a-eaf453caa1c4.png)

**Comment by cdwmhcc** [comment]
Agree

**Comment by rijkvanzanten** [comment]
Have you tried using the `image` interface rather than the `file` interface? 🙂

**Reply by lluishi93** [reply]
Thanks! Working like a charm

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
